docs/developers/protocol/technical-references/taking-and-making-offers/offer-list.md

```diff
@@ -5,97 +5,88 @@ sidebar_position: 2
-* Thus, since the ratio is in WETH/DAI, **the price is the `ratio`**
+Let's take a WETH/DAI market and look at its two corresponding offer lists, WETH-DAI and DAI-WETH, with `tickSpacing = 1`:
 
-We can illustrate this with the following sample DAI-WETH offer list with three offers. Only the main characteristics of the offers are shown (see the [offer data structure](reactive-offer/offer-data-structures.md#mgvlib-offer)).
+### Offer list #1: WETH-DAI ("asks")
 
+For a WETH/DAI market, the WETH-DAI offer list corresponds to the "asks" of the order book:
 
-|  Tick   | Ratio (WETH/DAI) | Offer ID | Gives (DAI) | Gas required | Maker Contract | Offer Gas Price |
-| ------- | ---------------- | -------- | ----------- | ------------ | -------------- | --------------- |
-| -79815  | 0.0003419        | 77       | 925.26      | 250,000      | 0x5678def      | 150             |
-|         |                  | 177      | 916.47      | 270,000      | 0x9101ghi      | 170             |   
-| -79748  | 0.0003442        | 42       | 871.76      | 300,000      | 0x1234abc      | 200             |
+* WETH is the _outbound_ token, i.e, offers give WETH
+* DAI is the _inbound_ token, i.e, offers want DAI
+* The unit of %%ratios|ratio%% is DAI/WETH and price = ratio.
 
+Here's an example of such an offer list:
 
-##### Understanding the table
-* **Tick**: a number derived from the ratio (price), pointing to a [tick bin](../tick-ratio.md#3-tick-bins). All offers in a bin have the same tick.
-* **Ratio**: tells us how much WETH (base) we get per DAI (quote).
-* **Offer ID**: more information [below](#offer-id).
-* **Gives**: more information on the [Ticks and ratio](../tick-ratio.md#price--wants) page.
-* **Gas required**: the amount of gas needed to cover all calls to the maker contract's [offer logic](./reactive-offer/maker-contract.md).
-* **Maker contract**: the address of the [maker contract](./reactive-offer/README.md) (smart offer).
-* **Offer gas price**: gas price override used to compute the order's %%provision|provision%% (see also [offer bounties](../taking-and-making-offers/reactive-offer/offer-provision.md#bounty-calculation)).
+|  Tick   | Ratio (DAI/WETH) | Offer ID | Gives (WETH) | Gas required | Maker Contract | Offer Gas Price |
+| ------- | ---------------- | -------- | -----------  | ------------ | -------------- | --------------- |
+| 75171   | 1838.53          | 42       | 0.7          | 220,000      | `0x2468xyz...` | 160             |
+|         |                  | 96       | 1.3          | 280,000      | `0x1357klm...` | 140             |
+| 75200   | 1843.87          | 7        | 0.6          | 210,000      | `0x3287opq...` | 190             |
 
+#### Understanding the table
 
-#### Offer list #2 - WETH-DAI
+* **Tick**: The discrete "price" %%tick|tick%% of the offer which corresponds to a ratio and price. Offers at the same tick are stored in FIFO order and in the table we only show the tick and ratio for the first offer with that tick.
```

Comment:
I would rather just show the tick and for all offers. Makes the table more readable ( tried to understand why information was missing) and makes the text a bit shorter here.

Reply:
ok. I would have preferred some kind of grouping of offers with  the same tick, but I don't see a good way of doing it with markdown. So I've made the change you suggested in [d6eab45](https://github.com/mangrovedao/mangrove-docs/pull/219/commits/d6eab45a10178a0391bf199c19540b99fe5a3459)